ingest/src/main/java/com/microsoft/azure/kusto/ingest/ResourceManager.java

```diff
@@ -91,6 +94,11 @@ public void close() {
         timer = null;
```

Comment:
I see you reverted also the `timer = null` that on the previous commit was after closing - I actually saw it and thought this is good so we don't get null pointer above